extensions/wikia/EditPageLayout/js/plugins/PageControls.js

```diff
@@ -436,15 +401,16 @@
 		renderPreview: function (extraData, type) {
 			var self = this;
 
-			require([ 'wikia.fluidlayout' ], function (fluidlayout) {
+			require([ 'wikia.breakpointsLayout' ], function (breakpointsLayout) {
 				var previewPadding = 22, // + 2px for borders
-					articleWidth = 660, width = articleWidth + (self.isGridLayout ? 30 : 0
-						), railBreakPoint = fluidlayout.getBreakpointSmall(), config = self.editor.config;
+					articleWidth = 660,
+					width = articleWidth + (self.isGridLayout ? 30 : 0),
```

Comment:
no, grid layout is something else (yes I know, yet another one...). You can see grid layout on http://wikia.com.
I created a ticket in our backlog to decide what to do with it.